gensim/test/test_ldamodel.py

```diff
@@ -265,6 +265,39 @@ def testGetDocumentTopics(self):
             self.assertTrue(isinstance(k, int))
             self.assertTrue(isinstance(v, float))
 
+    def testStaticTopics(self):
+
+        numpy.random.seed(0)
+        model = self.class_(self.corpus, id2word=dictionary, num_topics=2, passes= 100)
+
+        if isinstance(self.class_, ldamodel.LdaModel):
+            # check with id 
+            result = model.get_static_topic(2)
+            expected = [(1, 0.10658834157061754)]
+            self.assertEqual(result[0][0], expected[0][0])
+            self.assertAlmostEqual(result[0][1], expected[0][1])
+
+
+            # if user has entered word instead, check with word
+            result = model.get_static_topic(str(model.id2word[2]))
+            expected = [(1, 0.10658834157061754)]
```

Comment:
The precision of `0.10` is ok. That way multicore and singlecore models should have the same test.